Sources/DeepLearning/Layer.swift

```diff
@@ -1404,12 +1409,75 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNumeric
     }
 }
 
+/// A gated recurrent unit (GRU) cell.
+///
+/// Reference: [Gated recurrent unit](https://en.wikipedia.org/wiki/Gated_recurrent_unit).
+@_fixed_layout
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var resetWeight, updateWeight, inputHiddenWeight, hiddenHiddenWeight: Tensor<Scalar>
```

Comment:
```suggestion
    public var W_r, W_u, W_ih, W_hh: Tensor<Scalar>
```
In accordance with #113, we could just modify these here.